controllers/applicationvolumereplication_controller.go

```diff
@@ -22,13 +19,11 @@ import (
 	"fmt"
 	"reflect"
 	"sort"
-	"strings"
+	"time"
 
-	spokeClusterV1 "github.com/open-cluster-management/api/cluster/v1"
+	"github.com/ghodss/yaml"
```

Comment:
(nit) Although not a linter rule, the practice seems to  be to, split imports in 3 sections. Base (above, like "fmt"), External imports, Local imports. Removing a line space here would help keeping it as 3 (unnamed) sections.